synapse/cortex.py

```diff
@@ -410,6 +399,32 @@ def finiCortex():
 
         self.onfini(finiCortex)
 
+    def _initCoreLayers(self):
+        import synapse.cells as s_cells  # avoid import cycle
+        layersdir = pathlib.Path(self.dirn, 'layers')
+        layersdir.mkdir(exist_ok=True)
+        if pathlib.Path(layersdir, 'default').is_dir():
```

Comment:
lets add a `#pragma no:cover` here and a fixme to remove this function prior to 010 release.